.docker/entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+java -jar /opt/copybara/copybara_deploy.jar $COPYBARA_OPTIONS $COPYBARA_SUBCOMMAND $COPYBARA_CONFIG $COPYBARA_WORKFLOW $COPYBARA_SOURCEREF
```

Comment:
Could you add the Apache license header? (See other .sh files)

Reply:
Addressed in #45 , and added to the Dockerfile as well.